model/trace/gen-ai.yaml

```diff
@@ -56,32 +56,3 @@ groups:
           The `error.type` SHOULD match the error code returned by the Generative AI provider or the client library,
```

Comment:
> Should there be a way to know if a tool call was an error? How would we represent that?

Yes, the error is captured on the span, not inside the event. `error.type` attribute is a generic way to capture span errors.

If there is a need to add such attribute on the event, it can be done, but in case of LLM:
- when model responds, if there is a tool call there, then there is no error - response was successfully received. 
- when app executes a tool and returns the response back to genai, then if the request fails, it will fail later

I.e. I don't think there is a  need for `error.type` on specific events, but such attribute can be added if the need arrives